--- a/data/appdata/org.gnome.Recipes.appdata.xml.in	2019-09-13 23:34:10.181473486 +0200
+++ b/data/appdata/org.gnome.Recipes.appdata.xml.in	2019-09-13 23:36:06.623232101 +0200
@@ -450,16 +450,17 @@
     </release>
   </releases>
   <kudos>
-    <kudo>AppMenu</kudo>
     <kudo>HiDpiIcon</kudo>
     <kudo>ModernToolkit</kudo>
     <kudo>Notifications</kudo>
     <kudo>SearchProvider</kudo>
     <kudo>UserDocs</kudo>
   </kudos>
+  <content_rating type="oars-1.1" />
+  <launchable type="desktop-id">org.gnome.Recipes.desktop</launchable>
   <url type="homepage">https://wiki.gnome.org/Apps/Recipes</url>
   <url type="donation">https://www.gnome.org/friends</url>
-  <url type="bugtracker">https://bugzilla.gnome.org/browse.cgi?product=recipes</url>
+  <url type="bugtracker">https://gitlab.gnome.org/GNOME/recipes/issues/</url>
   <url type="translate">https://wiki.gnome.org/TranslationProject</url>
   <translation type="gettext">gnome-recipes</translation>
   <update_contact>matthias.clasen_at_gmail.com</update_contact>
